packages/color-area/src/spectrum-color-area.css

```diff
@@ -72,15 +72,6 @@ governing permissions and limitations under the License.
     -webkit-user-select: none;
     user-select: none;
 }
-:host([focused]) {
-    z-index: 2;
-}
```

Comment:
@pfulton some of the Color components remove this, and others do not. Which is correct?

Reply:
I believe @jenndiaz would know the answer to this better than I do!

Reply:
Are we referencing the z-index or the color handle focused styling? 

The color handle focused styling was all moved to be within the ColorHandle. 
For the z-index It looks like this one was removed by mistake, I've put up a PR to put this instance of it back.

Reply:
Specifically this `z-index`. If you're adding it back, we'll get that when it comes, just wanted to make sure whether this approach or the others was the right one.